modin/pandas/test/test_groupby.py

```diff
@@ -98,7 +98,7 @@ def wrapper(obj1, obj2, *args, **kwargs):
 
 @pytest.mark.parametrize("as_index", [True, False])
 def test_mixed_dtypes_groupby(as_index):
-    frame_data = np.random.randint(97, 198, size=(2**6, 2**4))
+    frame_data = np.random.RandomState(42).randint(97, 198, size=(2**6, 2**4))
```

Comment:
It was really hard to debug this test when the data is completely random, so decided to fix the seed

Reply:
Did you create an issue for completely random data? Looks loke a bug.

Reply:
No, because that's the only place where I noticed completely random data and fixed it right away